web-ts/package.json

```diff
@@ -48,6 +48,7 @@
     "moment": "2.29.1",
     "normalize-strings": "^1.1.1",
     "normalize.css": "^8.0.1",
+    "path-to-regexp": "^1.7.0",
```

Comment:
Do we use this or is this only used by react-router-dom generatepath?
Say: Do we have to import this our own?

Reply:
This is only used by `react-router-dom`, however there seems to be an issue that `<Redirect>` does not work without this:
https://github.com/ReactTraining/react-router/issues/6840#issuecomment-541039824